src/ansys/grantami/jobqueue/_connection.py

```diff
@@ -247,18 +247,14 @@ def jobs_where(
         list of AsyncJob
             List of jobs on the server matching the query.
         """
-        kwargs = {
-            "name_filter": name,
-            "job_type": job_type.value if job_type else None,
-            "status": status.value if status else None,
-            "description_filter": description,
-            "submitter_name_filter": submitter_name,
-        }
-
         with warnings.catch_warnings():
             warnings.simplefilter("ignore", UndefinedObjectWarning)
             filtered_job_resp = self.job_queue_api.get_jobs(
-                **{k: v for k, v in kwargs.items() if v is not None}
+                name_filter=name,
```

Comment:
mypy is failing when trying to validate the types on the `**{ ... }` statement, but it's not clear why. The change here doesn't make any changes to the types of the values, it only passes in `None` if the argument is `None`, instead of not providing it.

Either way, it seems to satisfy mypy, so we can go ahead with this change. The only reason for filtering out `None` arguments is if we were providing values to a PATCH operation, where there *is* a difference between providing None and not specifying a value. That isn't what's happening here, so this change seems safe.

Reply:
This is what I had to do on the release branch, it's a bit strange, but seems to be working